tools/reviewer/local_server/service/code_review.proto

```diff
@@ -130,4 +143,5 @@ message Comment {
   string created_by = 3;
   bool is_modified = 4;
   string id = 5;
+  int64 github_comment_id = 6;
```

Comment:
**oferb's comment:**
What is the `github_comment_id`? Is it coming from GitHub?

Reply:
Yes, it's ID of GitHub comment. This field will be set after the first sync with GitHub.